packages/api/cms-api/src/redirects/dto/redirect-input.factory.ts

```diff
@@ -22,7 +22,7 @@ export interface RedirectValidationArguments extends ValidationArguments {
 }
 
 export class RedirectInputFactory {
-    static create(linkBlock: Block): Type<RedirectInputInterface> {
+    static create({ linkBlock }: { linkBlock: Block }): Type<RedirectInputInterface> {
```

Comment:
This change looks like a leftover of a refactoring.

Reply:
this change was intentional, to have a similar signature for all factories.

and it's private